CoreInteractiveGraph/src/au/gov/asd/tac/constellation/graph/interaction/animation/FlyingAnimation.java

```diff
@@ -49,66 +49,75 @@
 
     @Override
     public void initialise(GraphWriteMethods wg) {
+        // dont initilise the animation if there is less than 2 vertexs
```

Comment:
vertexs -> nodes

Reply:
@antares1470 as in change the word to "nodes"? It's in code rather than GUI facing so I think its OK. Though grammatically it should be "vertices"